handlers/validator.go

```diff
@@ -683,16 +686,14 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 			lookback := int64(finalizedEpoch - (lastStatsDay+1)*utils.EpochsPerDay())
 			if lookback > 0 {
 				// logger.Infof("retrieving attestations not yet in stats, lookback is %v", lookback)
-				attestationsNotInStats, err := db.BigtableClient.GetValidatorAttestationHistory([]uint64{index}, finalizedEpoch-uint64(lookback), finalizedEpoch)
+				missedAttestations, err := db.BigtableClient.GetValidatorMissedAttestationHistory([]uint64{index}, finalizedEpoch-uint64(lookback), finalizedEpoch)
```

Comment:
While this change makes this part faster, it will not work properly once for example executed attestations have to be summed up too. What's your view on that?

Reply:
Then we would need another round of optimization ... but let's burn that bridge when we need it

Reply:
Okay

Reply:
Sure.